### PR TITLE
added dropdown with 7d, 15d and 30d values

### PR DESCRIPTION
**Pull request by Anikkk**
image

### CODE REVIEW

**Comment by kaivan-s** [comment]
As discussed, please remove the micro and macro toggle as well.
Still there as per the screenshot.

File: src/Components/ChartsComponent.js [hunk 1]
```diff
@@ -36,12 +42,12 @@ const ChartComponent = () => {
         let response = ''
         if (isMacro) {
           response = await fetch(
-            `${apiUrl}/get-forecast/${selectedCommodity}/macro`
+            `${apiUrl}/get-forecast/${selectedCommodity}/macro/`
           );
         }
```
Comment (kaivan-s) [inline src/Components/ChartsComponent.js:47]: As discussed, please remove this. For now we are just using the micro part.

**Comment by kaivan-s** [comment]
Also, remove the italics part from both the boxes below i.e. the summary boxes.

**Comment by kaivan-s** [comment]
If possible, please add a GIF to see how the functionality is working after making the changes in the dropdown.

File: src/Components/ChartsComponent.js (hunk 1 above)
Comment (Anikkk) [reply src/Components/ChartsComponent.js]: did that changes

**Review by kaivan-s — APPROVED** [review]

**Pull request merged**